arangod/RestServer/Metric/Batch.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
```

Comment:
By convention, we place the `#pragma once` below the copyright header. I would favor doing it in this PR too.

Reply:
Ok